src/test/e2e/30_config_file_test.go

```diff
@@ -12,7 +13,7 @@ import (
 
 func TestConfigFile(t *testing.T) {
 	t.Log("E2E: Config file")
-	e2e.setup(t)
+	e2e.setupWithCluster(t)
```

Comment:
As a general practice, we've tried to put as many tests as we can without needing a cluster for faster feedback. Is there any way to accomplish this test without needing a cluster? If you feel we need it to test templating in-cluster, than we can leave it.

Reply:
I didn't notice before that we had this as a < 20 test on my prior review

Reply:
I would really like to test with templating since this bug would not be caught otherwise and could be easy to reintroduce through a refactor if someone was not careful.  (and while this test won't run on Windows now, the time cost wasn't that significant to have the cluster).

In the future we could template `file` includes as well and refactor this and the package variables test.  I'll create an issue for that.